cartographer/mapping_3d/motion_filter.cc

```diff
@@ -39,8 +39,9 @@ MotionFilter::MotionFilter(const proto::MotionFilterOptions& options)
 
 bool MotionFilter::IsSimilar(const common::Time time,
                              const transform::Rigid3d& pose) {
-  LOG_EVERY_N(INFO, 500) << "Motion filter reduced the number of scans to "
-                         << 100. * num_different_ / num_total_ << "%.";
+  LOG_IF_EVERY_N(INFO, num_total_ > 0, 500)
```

Comment:
This logs after the first call which is still not very useful information. `LOG_IF_EVERY_N(INFO, num_total_ >= 500, 500)`? Then we would skip the NaN we log now, but leaves the logging unchanged otherwise.